roles/openshift_examples/tasks/main.yml

```diff
@@ -36,6 +36,7 @@
   unarchive:
     src: "{{ copy_examples_mktemp.stdout }}/openshift-examples.tar"
     dest: "{{ examples_base }}/"
+    mode: 0755
```

Comment:
@tbielawa what permissions do the files in the directory have, though? My comment about using the symbolic permissions was to prevent the files from potentially getting execute permissions, but still allow setting execute for the directories.